server/test/com/xsn/explorer/data/BlockPostgresDataHandlerSpec.scala

```diff
@@ -215,6 +216,29 @@ class BlockPostgresDataHandlerSpec extends PostgresDataHandlerSpec with BeforeAn
     }
   }
 
+  "get blockhead" should {
```

Comment:
if you look carefully, this is usually named as the method that it is testing, `getHeader` in this case.